src/ucm/ucx-ucm.pc.in

```diff
@@ -0,0 +1,12 @@
+prefix = @prefix@
```

Comment:
Pls add NVIDIA copyright in all pkg-config files

Reply:
> Pls add NVIDIA copyright in all pkg-config files

added